### PR TITLE
Follow patterns of the project.

### PR DESCRIPTION
**Pull request by Fizzify**
Fix an issue in the _app.tsx where it would give this error. This is because the curly braces aren't needed, it is already getting it from "next/app", so you wouldn't need the curly braces anymore.

Before changes:
image

After changes:
image

### CODE REVIEW

**Comment by juliusmarminge** [comment]
The base app is created using `create-next-app`, see _app file here: https://github.com/vercel/next.js/blob/canary/packages/create-next-app/templates/typescript/pages/_app.tsx

I usually do: 
```ts
import type { AppType } from "next/dist/shared/lib/utils";

const MyApp: AppType = ({ Component, PageProps }) => { ... }
export default MyApp
```

and this is how we have in all other _app.tsx so maybe migrate that pattern here too?

**Comment by Fizzify** [comment]
Ohhh wait I get what you said now.

File: template/base/src/pages/_app.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import "../styles/globals.css";
-import type { AppProps } from "next/app";
+import type { AppProps } from "next/dist/shared/lib/utils";
 
 function MyApp({ Component, pageProps }: AppProps) {
```
Comment (juliusmarminge) [inline template/base/src/pages/_app.tsx:4]: I was actually thinking of making this an arrow function like we have in most other places. 
```ts
import type { AppType } from "next/dist/shared/lib/utils";

const MyApp: AppType = ({ Component, pageProps }) => { ... }
export default MyApp;
```

What do you think?

File: template/page-studs/_app/with-auth.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import "../styles/globals.css";
-import type { AppProps } from "next/app";
+import type { AppProps } from "next/dist/shared/lib/utils";
 import { SessionProvider } from "next-auth/react";
 
 function MyApp({ Component, pageProps: { session, ...pageProps } }: AppProps) {
```
Comment (juliusmarminge) [inline template/page-studs/_app/with-auth.tsx:5]: Same as above

File: template/base/src/pages/_app.tsx (hunk 1 above)
Comment (Fizzify) [reply template/base/src/pages/_app.tsx:4]: Yeah, I think it will be cool, so that the code will be consistent throughout the project.

**Comment by Fizzify** [comment]
Alright I made the PR to the main branch.

File: template/base/src/pages/_app.tsx (hunk 1 above)
Comment (juliusmarminge) [reply template/base/src/pages/_app.tsx:4]: > If you want then I can just add those changes to this PR as well.

Yes, please do!

**Pull request merged**